.circleci/unittest/ios/scripts/binary_ios_build.sh

```diff
@@ -22,6 +22,8 @@ export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
 
 # sync submodules
 cd ${PROJ_ROOT_IOS}
+git fetch
+git checkout release/0.13
```

Comment:
I missed this. I think you are right, doesn't matter to which branch you will send the PR. Thanks for clarifying.